notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
     "\n",
```

Comment:
<p>What about these graphs tells you that there is positive correlation between size, changed_file_number and body size, but not time to merge? Its often good practice to assume your audience does not necessarily have the same statistics background as you do and provide some justification to your claims about the data.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/289/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>